share/pot/prune.sh

```diff
@@ -7,52 +7,80 @@ prune-help()
 	echo '  -h print this help'
 	echo '  -v verbose'
 	echo '  -q quite - prune with no output'
+	echo '  -g grace period - do not prune pots that just finished executing'
 	echo '  -n dry-run - do not destroy anything'
 }
 
 # $1 pot name
 _prune_pot()
 {
-	local _pname _quiet _dry_run
+	local _pname _quiet _dry_run _grace_period _confdir
 	_pname=$1
 	_dry_run=$2
 	_quiet=$3
+	_grace_period=$4
+	_confdir="${POT_FS_ROOT}/jails/$_pname/conf"
+
 	if ! _is_pot_running "$_pname" ; then
```

Comment:
@pizzamig A lot of the changes in this function is changing from a nested-if- to barrier-style to reduce indentation. Functional changes are about grace_period.

Reply:
@grembo I've noticed the re-organization and I agree with anything that improve the readability and the maintainability of the code! Thanks!